tests/remotes/__init__.py

```diff
@@ -33,3 +43,26 @@
     ssh_remote,
     ssh_server,
 )
+
+
+@pytest.fixture
+def workspace(tmp_dir, dvc, request):
```

Comment:
will introduce a similar `remote` fixture as well instead of current explicit ones.

Reply:
Also, this really illustrates the need for https://github.com/iterative/dvc/issues/3920